elements/sdk-build-depends/cython.bst

```diff
@@ -0,0 +1,9 @@
+kind: pip
+
+build-depends:
+- freedesktop-sdk.bst:public-stacks/buildsystem-python.bst
+
+sources:
+- kind: git_tag
+  url: github_com:cython/cython
+  track: 0.29.13
```

Comment:
This seems like something we might want to add to the FDO Sdk. :thinking: 

Did you ask over there?

Reply:
That makes sense! Feature request with a patch filed upstream: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/issues/910